app/controllers/api/v0/followers_controller.rb

```diff
@@ -4,28 +4,18 @@ class FollowersController < ApiController
       before_action :authenticate_with_api_key_or_current_user!
       before_action -> { limit_per_page(default: 80, max: 1000) }
 
-      def organizations
```

Comment:
👍 I really like this, while the API is v0 we shouldn't be afraid of this sort of breaking change, especially since you verified that it's not been used recently.